### PR TITLE
read/write tab icons

### PR DESCRIPTION
**Pull request by xtine**
N&C style override to add svg icons to read/write tabs. #36

screen shot 2016-04-04 at 12 41 20 pm

### CODE REVIEW

**Comment by cmc333333** [comment]
I may be misreading, but it looks like there's a separate SVG file which isn't used and then two inline-SVGs. Should these be separate files (allows browsers to cache the content, but difficult to color via CSS)? If not, is there any reason to have them only in eregs/n&c? Presumably we could embed the icons in -site and color them in the ereg/n&c override?

File: notice_and_comment/static/regulations/css/less/comment-custom.less [hunk 1]
```diff
@@ -8,26 +9,55 @@
...
+    span {
+      padding-left: 5px;
+    }
+  }
+
+  .active-mode {
+
+    .read-icon,
+    .write-icon {
+      fill: @pacific;
```
Comment (donjo) [inline notice_and_comment/static/regulations/css/less/comment-custom.less]: this fill should be `@blue` I believe. In general that's what I've been overriding any instances of `@pacific` from -site with

**Comment by donjo** [comment]
screen shot 2016-04-04 at 4 20 04 pm

there's a little 1px overhang on these tabs that may have resulted from our top nav going from 60px in -site to 56px in notice-and-comment. Adding `height: 33px` to `.preamble-header .read-proposal, .preamble-header .write-comment {` is one option to fix. The other being re-calculate some of the heights on `.sub-head` and others that are defined in `em` units

I'm also thinking of maybe 2 other things, maybe making the tabs a little bit tighter (225px width instead of 245px) and using the semibold (weight 600) font. Ex:

screen shot 2016-04-04 at 4 34 11 pm

Any thoughts @jehlers?

**Comment by xtine** [comment]
I removed `write-icon.svg` because I forgot to earlier.

**Comment by xtine** [comment]
I do prefer having svgs inline especially if they are small since it's one less request and asset to take care of. I could go either way in having it in -site instead, just thought of it being a n&c theme at first.

**Comment by cmc333333** [comment]
@dojo should we make an issue for that?

**Comment by cmc333333** [comment]
Polish in #97

**Pull request merged**